design-toolkit/components/src/components/slider/range-slider.stories.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import type { Meta, StoryObj } from '@storybook/react';
+import { RangeSlider } from './index';
+
+const meta: Meta<typeof Range> = {
+  title: 'Components/RangeSlider',
+  component: RangeSlider,
+  args: {
+    orientation: 'vertical',
+    layout: 'inline',
```

Comment:
```suggestion
    orientation: 'horizontal',
    layout: 'stacked',
```
same comment here since this configuration will be used the most often.